documentation/app/templates/components/list-item-template.hbs

```diff
@@ -0,0 +1,856 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<PageHeader
+  @title="List Item Template"
+  @figma="https://www.figma.com/file/SenC8kTQNUJKCvOvcL8WPu/Consul-Component-Library?type=design&node-id=722-120845&t=YhNf39KC0C4MRjwb-0"
+  @github="https://github.com/hashicorp/consul-ui-toolkit/blob/main/toolkit/src/components/cut/list-item/template/index.hbs"
+  @status="development"
+>
+  A List Item Template is used as a layout template to create a number of Consul
+  feature specific list items. It can also be used as a template for other items
+  that are able to make use of the template layout and its available
+  pre-determined element options.
```

Comment:
Update copy to: "The List Item Template is meant to be used as a reusable pattern to create both visually and functionally  consistent List Items. This component is primarily used as a template to create List Items for most Consul specific features."